10.9-libcxx/stable/main/finkinfo/libs/rmods/cran-vgam-r.info

```diff
@@ -68,7 +68,9 @@ estimation (MLE) or penalized MLE. Also fits constrained ordination
 models in ecology.
 <<
 DescPackaging: <<
-  R (>= 3.4.0)
+  Package version   R version
+  1.1-5             >= 3.5.0
```

Comment:
This says 1.1-5 is for R >= 3.5.0, but cran-vgam-r.info still shows it available for R-3.4 in `Type` field.